src/fu-util.c

```diff
@@ -3484,6 +3524,16 @@ fu_util_security(FuUtilPrivate *priv, gchar **values, GError **error)
 	if (priv->no_unreported_check)
 		return TRUE;
 
+	/* any things we can fix? */
+	for (guint j = 0; j < attrs->len; j++) {
+		FwupdSecurityAttr *attr = g_ptr_array_index(attrs, j);
+		if (fwupd_security_attr_get_bios_attr_id(attr) != NULL &&
+		    fwupd_security_attr_get_bios_attr_value(attr) != NULL) {
+			if (!fu_util_security_modify_bios_attr(priv, attr, error))
+				return FALSE;
+		}
+	}
+
 	/* upload, with confirmation */
```

Comment:
the upload should come before the BIOS setting change.

Reply:
The upload should come on the next reboot, i.e. not this boot tbh

Reply:
But don't you want to see data that someone had a fail, and then see MORE data if they fixed it?

Reply:
The new one would override the old one, for the same machine id.